### PR TITLE
switching to community creates explicit subscription

### PR DESCRIPTION
**Pull request by nickpell**
also makes sure that invoices will be skipped for these community subscriptions

@proteusvacuum cc: @snopoke

### CODE REVIEW

**Comment by proteusvacuum** [comment]
:boot: :construction_worker: as the test failure looked unrelated

File: corehq/apps/accounting/tasks.py [hunk 1]
```diff
@@ -547,43 +548,46 @@ def update_exchange_rates(app_id=settings.OPEN_EXCHANGE_RATES_API_ID):
...
+        CONSISTENT_DATES_CHECK
+    ).filter(
+        Q(date_end__gt=from_date) | Q(date_end__isnull=True),
+        date_start__lte=from_date,
+        subscriber__domain=domain_name,
+    ).exists():
+        assign_explicit_community_subscription(domain_name, from_date)
+
+
+def assign_explicit_community_subscription(domain_name, start_date):
```
Comment (proteusvacuum) [inline corehq/apps/accounting/tasks.py]: should this ever get called directly? Or should we always call `ensure_explicit_community_subscription`?

Comment (proteusvacuum) [inline corehq/apps/accounting/tasks.py]: (if so might want to make this private)

Comment (nickpell) [inline corehq/apps/accounting/tasks.py]: going to call this directly in an upcoming PR...

**Comment by nickpell** [comment]
✅

File: corehq/apps/accounting/tasks.py (hunk 1 above)
Comment (proteusvacuum) [inline corehq/apps/accounting/tasks.py]: :+1:

**Pull request merged**